karma.conf.js

```diff
@@ -16,7 +16,7 @@ module.exports = function(config) {
     // with the preprocessors, to watch the source files and serve the compiled
     // files.
     files: [
-      'jujugui/static/gui/build/app/assets/javascripts/jujulib/*.js',
```

Comment:
Without this change, any code updates in jujulib would need a `make clean` in order to be picked up by a test run.